rest/src/main/java/org/jboss/pnc/rest/JaxRsActivatorNew.java

```diff
@@ -218,13 +220,15 @@ private void addEndpoints(Set<Class<?>> resources) {
         resources.add(UserEndpointImpl.class);
 
         resources.add(BuildRecordAliasEndpointImpl.class);
+        resources.add(DeliverableAnalyzerReportEndpointImpl.class);
 
         resources.add(HealthCheckEndpointImpl.class);
         resources.add(GenericSettingEndpointImpl.class);
         resources.add(CacheEndpointImpl.class);
         resources.add(DebugEndpointImpl.class);
         resources.add(DeliverableAnalysisEndpointImpl.class);
         resources.add(OperationEndpointImpl.class);
+        resources.add(DeliverableAnalyzerReportRepositoryImpl.class);
```

Comment:
why?

Reply:
Oooh yes, we really dont need to define `DeliverableAnalyzerReportRepostoryImpl` bean as one of the endpoints implementation beans :<)
dbd491b